aws-core/src/main/java/com/amplifyframework/auth/AWSAuthSessionInternal.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
FWIW, I think our standard is to adopt the no-year version of the copyright message. See examples at https://github.com/aws-amplify/amplify-swift/blob/main/Amplify/Amplify.swift#L1-L6, enforced by https://github.com/aws-amplify/amplify-swift/blob/main/.swiftformat#L12-L13

Reply:
Great! I'll create a ticket for us to do in a future PR.